code/game/objects/items/weapons/shields.dm

```diff
@@ -150,6 +150,11 @@
 	origin_tech = "{'materials':4,'magnets':3,'esoteric':4}"
 	attack_verb = list("shoved", "bashed")
 	var/active = 0
+	var/shield_light = "#006aff"
```

Comment:
It would probably be better to name this `shield_light_color`, at a glance this looks like it would set light range.